package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "col-league",
+  "version": "0.0.1",
+  "description": "A tool for employees to learn the names of their fellow colleagues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/excellaco/face_it.git"
+  },
+  "contributors": [
+    {
+      "name": "Emmanuel Apau",
+      "email": "etapau@gmail.com",
+      "url": "https://github.com/kave"
+    },
+    {
+      "name": "Sean Lewis",
+      "email": "",
+      "url": "https://github.com/madridsoccer5 "
```

Comment:
@madridsoccer5 hey after you merge this don't forget to put in your email in your next pull request